templates/views/account/changeUsername.pug

```diff
@@ -8,6 +8,7 @@ block content
             .col-md-12
                 h1.account-title Change Username
                 h4.account-subtitle.text-center Change your username by using the form below. Once it is changed, you can log in to your account with the new username.
+                h4.account-subtitle.text-center You can only change the name every 30 days and your old username will be reserved for 6 months. In case you choose an offending username you might get banned for 30 days.
```

Comment:
We don't have exact rules about banning periods. Just write you might get banned